backend/src/index.js

```diff
@@ -4,17 +4,13 @@ const cookieParser = require('cookie-parser')
 const logger = require('morgan')
 const app = express()
 
-if (app.get('env') !== 'test') app.use(logger('dev'))
 app.use(express.json())
 app.use(express.urlencoded({ extended: false }))
 app.use(cookieParser())
 app.use(express.static(path.join(__dirname, '../public')))
 if (app.get('env') === 'development') {
-    app.use(function (_, res, next) {
-        res.header('Access-Control-Allow-Origin', '*')
-        res.header('Access-Control-Allow-Headers', 'Origin, X-Requested-With, Content-Type, Accept')
-        next()
-    })
+    app.use(logger('dev'))
+    app.use(require('./utils/cors').cors)
```

Comment:
This is a good idea to differentiate dev and prod.

Reply:
👍